### ISSUE TITLE
Method for getting direct HashMap

### ISSUE THREAD

**Issue by StiviiK** [DeBukkIt/SimpleFileStorage#1]
A Method for getting the HashMap would be useful. 
Example:

``` java
HashMap<String, Object> List = fs.getAll();
List.forEach(
   (k, v) -> {
      System.out.println(k + ", " + v);
   }
);
```

You could split it up like this:

``` java
/**
* All stored objects in an ArrayList of Objects
* @return all stored objects in an ArrayList of Objects
*/
public ArrayList<Object> getAllObjects(){
   ArrayList<Object> result = new ArrayList<Object>();
   for(Object c: storageMap.values()){
      result.add(c);
   }
   return result;
}

/**
* All stored objects and keys in an HashMap of String and Objects
* @return all stored objects and keys in an HashMap of String and Objects
*/
public HashMap<String, Object> getAll(){
   return storageMap;
}
```

(This is to short for a Fork + Pull request)

**Comment by DeBukkIt** [comment]
Thank you for these ideas! They have been implemented by commit 7aa39a95580f78203b61d8f536518ca6284a446f some seconds ago with credits to you.

**Comment by StiviiK** [comment]
:+1:
